NAMESPACE

```diff
@@ -119,6 +120,16 @@ export(uniqueEntities)
 export(updatePKParameter)
 export(validateDimension)
 export(validateUnit)
+import(ospsuite.utils)
 import(rClr)
 import(stringr)
+importFrom(ospsuite.utils,"%||%")

```

Comment:
Why is this required? (e.g. why is line 123 not enough?)

Reply:
This brings the `NAMESPACE` of `{ospsuite.utils}` in `{ospsuite}` package, so that you can use exported functions inside this package, BUT these functions will not be available to the user (because they are not exported).

```r
import(ospsuite.utils)
```

In order to re-export a function, it first needs to explicitly imported and then exported. This is why you see `importFrom` tags **only** for those functions which need to be exported:

https://github.com/Open-Systems-Pharmacology/OSPSuite-R/blob/006c31500d19a03f51bfd9d294dfcac3ae263209/NAMESPACE#L126-L134)

Reply:
ok. I don't think we should export those methods at all. But this is not very important. Merging